language/tools/vm-genesis/src/lib.rs

```diff
@@ -519,6 +525,7 @@ pub fn generate_genesis_change_set_for_testing(stdlib_options: StdLibOptions) ->
     let swarm = libra_config::generator::validator_swarm_for_testing(10);
 
     encode_genesis_change_set(
+        &GENESIS_KEYPAIR.1,
         &GENESIS_KEYPAIR.1,
```

Comment:
Is the treasury and libra root always the same?

Reply:
in our tests, unfortunately... if I have time, I'll update smoketest so that it can use two keys that way we get more confidence before next week... but I'll probably delay on this until later...